packages/client/base/src/services/hosted/v3/crossmintHostedCheckoutV3Service.ts

```diff
@@ -27,37 +27,37 @@ export function crossmintHostedCheckoutV3Service({
         return `${urlWithPath}?${queryParams.toString()}`;
     }
 
-    function createPopupClient() {
-        const url = getUrl(hostedCheckoutProps);
+    function createPopupClient(url: string) {
         return PopupWindow.initSync(url, {
             width: 450,
             height: 750,
             crossOrigin: true,
         });
     }
 
-    // TODO: Implement new tab client
-    function createNewTabClient(): ReturnType<typeof PopupWindow.initSync> {
-        throw new Error("Not implemented");
+    function createNewTabClient(url: string): ReturnType<typeof NewTabWindow.initSync> {
+        return NewTabWindow.initSync(url, {});
     }
 
     // TODO: Implement same tab client
```

Comment:
can we remove todo?

Reply:
thats on the next if, which isnt implemented yet